packages/harpstrata/src/get-harp-strata/utils/get-active-ids-pair/get-active-ids-pair.ts

```diff
@@ -7,8 +7,14 @@ import {
 } from '../get-counterpart-active-ids'
 import type { IsActiveProps, ActiveIdsPair } from '../../../types'
 
+// TODO: This function definitely need to deal with both of the harpface matrices
```

Comment:
Big TODO here I think. There is the possibility that we will not get a complete list of active id's to convert or persist in to the next harpstrata unless we observe both of the harpfaces. I think that having a good think about what the type of the `harpface2` property should be will be key here and in other places to make things as slick as possible.

Reply:
On closer inspection this isn't as much of a problem as it first seemed. It could use fixing, but it will not represent a significant risk the way things are setup at the moment. See raised issue: https://github.com/js-jslog/harpguru/issues/136